application/src/test/java/org/thingsboard/server/service/queue/ruleengine/TbRuleEngineQueueConsumerManagerTest.java

```diff
@@ -459,7 +453,7 @@ public void testDelete_consumerPerPartition() {
         int msgCount = totalConsumedMsgs.get();
 
         await().atLeast(2, TimeUnit.SECONDS) // based on topicDeletionDelayInSec(5) = 5 - ( 3 seconds the code may execute starting consumerManager.delete() call)
-                .atMost(7, TimeUnit.SECONDS)
+                .atMost(10, TimeUnit.SECONDS)
```

Comment:
It would be nice to have a constant TIMEOUT with a lot bigger value. E.g. 30 sec. Some build agents on shared resources can be extremely busy sometimes and the test can fail due to temporary build machine resource starvation.